usaspending_api/disaster/v2/views/overview.py

```diff
@@ -55,26 +55,29 @@ def funding(self):
             .annotate(
                 def_code=F("disaster_emergency_fund_code"),
                 amount=Sum("total_budgetary_resources_cpe"),
-                unobligated_balance=Sum("budget_authority_unobligated_balance_brought_forward_cpe"),
+                total_budget_authority=(
+                    Sum("total_budgetary_resources_cpe") - (
+                        Sum("budget_authority_unobligated_balance_brought_forward_cpe") +
+                        Sum("deobligations_or_recoveries_or_refunds_from_prior_year_cpe") +
+                        Sum("prior_year_paid_obligation_recoveries")
+                    )
+                )
             )
-            .values("def_code", "amount", "unobligated_balance")
+            .values("def_code", "amount", "total_budget_authority")
         )
 
-        total_budget_authority = self.sum_values(funding, "amount") - self.sum_values(funding, "unobligated_balance")
+        total_budget_authority = self.sum_values(funding, "total_budget_authority")
 
         for entry in funding:
-            del entry["unobligated_balance"]
+            del entry["total_budget_authority"]
```

Comment:
Makes me curious if we want the amounts for each DEFC to be the same TBR calculation instead of `total_budgetary_resources_cpe`?

Reply:
Follow-up: I can't think of why the individual DEFC Code amounts should't add up to the TBR reported by the endpoint. They originally were the same until #2873